src/helpers/divideByFilteredString.js

```diff
@@ -1,5 +1,7 @@
-export const divideByFilteredString = (inputText, searchedText) => {
+export const divideByFilteredString = (inputText, inputSearched) => {
+  if (inputSearched === null || inputSearched === undefined || inputSearched === '') return [];
```

Comment:
`if (inpuSearched)` should be enough. 
to be super explicit two first conditions can be changed to `inputSearched == null` (coersion will do also validation to undefined.